libraries/lib-graphics/CMakeLists.txt

```diff
@@ -5,6 +5,12 @@ A library responsible for custom rendering in Audacity
 set( SOURCES
    FrameStatistics.cpp
    FrameStatistics.h
+
+   graphics/Color.cpp
+   graphics/Color.h
```

Comment:
I would like to revisit at some point and check, which types are actually used. As Paul mentioned, we need some fixes and unit tests for Rect classes.

Reply:
Judging by the next commit we need Color, Rect, Point and Size, but we don't need most of the functions except for basic math